### PR TITLE
Move all logging to use static Logging class.

### PR DESCRIPTION
**Pull request by benwoo1110**
Same as https://github.com/Multiverse/Multiverse-Core/pull/2458.

### CODE REVIEW

File: src/main/java/com/onarandombox/MultiverseNetherPortals/utils/MVNameChecker.java [hunk 1]
```diff
@@ -66,7 +67,7 @@ public String getNetherName(String normalName) {
 	public String getEndName(String normalName) {
         final String endName = this.plugin.getEndPrefix() + normalName + this.plugin.getEndSuffix();
         if (plugin.getCore().getMVWorldManager().isMVWorld(endName)) {
-            this.plugin.log(Level.FINEST,  "Selected end world '" + endName + "' for normal '" + normalName + "'");
+			Logging.finest("Selected end world '" + endName + "' for normal '" + normalName + "'");
```
Comment (nicegamer7) [inline src/main/java/com/onarandombox/MultiverseNetherPortals/utils/MVNameChecker.java:70]: Using tabs here for some reason.

File: src/main/java/com/onarandombox/MultiverseNetherPortals/utils/MVNameChecker.java [hunk 2]
```diff
@@ -102,7 +103,7 @@ public String getNormalName(String netherName, PortalType type) {
             }
         }
         if (!normalName.equals(netherName) && plugin.getCore().getMVWorldManager().isMVWorld(normalName)) {
-            this.plugin.log(Level.FINEST,  "Selected normal world '" + normalName + "' for " + type + " '" + netherName + "'");
+			Logging.finest("Selected normal world '" + normalName + "' for " + type + " '" + netherName + "'");
```
Comment (nicegamer7) [inline src/main/java/com/onarandombox/MultiverseNetherPortals/utils/MVNameChecker.java:106]: Same as above.

**Review by nicegamer7 — CHANGES_REQUESTED** [review]
Looks good, just need to change tabs to spaces.

File: src/main/java/com/onarandombox/MultiverseNetherPortals/utils/MVNameChecker.java (hunk 2 above)
Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseNetherPortals/utils/MVNameChecker.java]: can seem to change it... is intelliJ being dumb lol

Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseNetherPortals/utils/MVNameChecker.java]: should be fixed lol idk why the entire file was in tabs not spaces.

**Comment by nicegamer7** [comment]
Could you combine the two tab commits? Or open a separate PR and then rebase. Or I can do it if you want, let me know.

**Comment by benwoo1110** [comment]
Alright should be good now.

**Comment by nicegamer7** [comment]
I guess I should've clarified...

I meant keep the tab commit separate from the logging commit. What I meant by combine was combine the two tab commits.

**Comment by benwoo1110** [comment]
ah rip ok lol

**Comment by benwoo1110** [comment]
yep should be it haha

**Review by nicegamer7 — APPROVED** [review]
Perfect 👍.

**Pull request merged**